clvm/chia_dialect_constants.py

```diff
@@ -0,0 +1,43 @@
+from .casts import int_to_bytes
+
+KEYWORDS = (
+    # core opcodes 0x01-x08
+    ". q a i c f r l x "
+
+    # opcodes on atoms as strings 0x09-0x0f
+    "= >s sha256 substr strlen concat . "
+
+    # opcodes on atoms as ints 0x10-0x17
+    "+ - * / divmod > ash lsh "
+
+    # opcodes on atoms as vectors of bools 0x18-0x1c
+    "logand logior logxor lognot . "
+
+    # opcodes for bls 1381 0x1d-0x1f
+    "point_add pubkey_for_exp . "
+
+    # bool opcodes 0x20-0x23
+    "not any all . "
+
+    # misc 0x24
+    "softfork "
+).split()
+
+KEYWORD_FROM_ATOM = {int_to_bytes(k): v for k, v in enumerate(KEYWORDS)}
+KEYWORD_TO_ATOM = {v: k for k, v in KEYWORD_FROM_ATOM.items()}
+
+KEYWORD_TO_LONG_KEYWORD = {
```

Comment:
Can you build this table algorithmically from `OP_REWRITE`?

Reply:
Or vice-versa?

Reply:
I'm not sure this constant is actually Chia dialect-specific.

Reply:
I included it there mainly to avoid another layer when resolving dependencies, but it does make sense to put this on a layer in between.